cypress/e2e/bookStore.cy.js

```diff
@@ -1,11 +1,40 @@
 /// <reference types='cypress' />
 
+const { generateData } = require('../support/generate');
+
 describe('Book Store app', () => {
-  before(() => {
-    
+  beforeEach(() => {
+    cy.visit('/login');
+    cy.login();
+    cy.visit('/profile')
+  });
+
+  it('should check if the user is logged in', () => {
```

Comment:
you should check it via UI since it's a test for login. in all other tests where you just need to be logged in to check smth else, you should use `login()` command